pkg/kudoctl/cmd/generate/maintainer_test.go

```diff
@@ -0,0 +1,57 @@
+package generate
+
+import (
+	"flag"
+	"io/ioutil"
+	"path/filepath"
+	"testing"
+
+	"github.com/spf13/afero"
+	"github.com/stretchr/testify/assert"
+
+	"github.com/kudobuilder/kudo/pkg/apis/kudo/v1beta1"
+	"github.com/kudobuilder/kudo/pkg/kudoctl/files"
+)
+
+var updateGolden = flag.Bool("update", false, "update .golden files and manifests in /config/crd")
+
+func TestAddMaintainer(t *testing.T) {
+	goldenFile := "maintainer"
+	path := "/opt/zk"
```

Comment:
stylistic nit, wdyt about moving this declaration closer to it's first use on line 28? same with goldenFile